src/core/core.scss

```diff
@@ -0,0 +1,3 @@
+main {
```

Comment:
Thanks for the explanation! Sounds sensible. Let's keep it like that we can always refactor the code in the future